clustermesh/clustermesh.go

```diff
@@ -1587,7 +1587,7 @@ if [ "$1" = "uninstall" ] ; then
     fi
     if [ -f /usr/bin/cilium ] ; then
         echo "Removing /usr/bin/cilium"
-        ${SUDO} rm /usr/bin/cilium
+        ${SUDO} rm /usr/bin/cilium /usr/bin/cilium-dbg
```

Comment:
The `if` statement above only checks for the existence of `/usr/bin/cilium`, so this will fail to uninstall older installations which didn't create the `cilium-dbg` binary. I guess it should be moved to a separate `if` statement.

Reply:
Ah I missed that this was not doing a force deletion. :+1: